src/components/thumbnail/class-work.tsx

```diff
@@ -20,13 +40,44 @@ export class ClassWorkComponent extends BaseComponent<IProps, {}> {
     sections.forEach((section) => {
       publications.push(...documents.getLatestPublicationsForSection(section.id, this.stores.class));
     });
+    const sectionTypes: any = [];
+    each(SectionType, type => {
+      sectionTypes.push(type);
+    });
```

Comment:
Is there a cleaner way to do this?  I just want those sectionTypes in an array so I can render using map.

Reply:
Yes!  That works well and reduces this to a single line.